packages/nimble-components/src/combobox/tests/combobox.spec.ts

```diff
@@ -416,7 +475,7 @@ describe('Combobox', () => {
             element.value = 'two';
             await connect();
 
-            expect(element.value).toBe('Two');
+            expect(element.value).toBe('two');
```

Comment:
You might even consider removing the `value` setters in the template as they are unused by the `Combobox`.